### ISSUE TITLE
[Feature Request]: Adding new tenants related resources for Grafana UI Enterprise edition

### ISSUE THREAD

**Issue by roman-k3i** [grafana/terraform-provider-grafana#1765]
### Feature Request

That would be great if you can add the following terraform resources for Grafana Enterprise Edition: `grafana_enterprise_tenant`, `grafana_enterprise_access_policy`, `grafana_enterprise_access_policy_token`. Currently we manage it through Grafana UI plugins API with Admin token. We would like to manage it by terraform to store state of these resources. Will be much obliged for such additional functionality for Grafana Enterprise edition.

**Comment by Duologic** [comment]
Is this the feature that you are talking about? https://grafana.com/docs/enterprise-metrics/latest/tenant-management/

**Comment by roman-k3i** [comment]
@Duologic Yes you are right. To have tenant based resources(and tenant itself) of Enterprise Logs, Traces and Metrics.
Our team currently provision through Admin HTTP API resources and having those resources managed by terraform would extremely simply our lives and we will have states of this resources.

**Comment by vtorosyan** [comment]
@roman-k3i Sorry it is still a bit unclear to me what is the need, we have resources for [cloud access policy](https://registry.terraform.io/providers/grafana/grafana/latest/docs/resources/cloud_access_policy) and [access policy token](https://registry.terraform.io/providers/grafana/grafana/latest/docs/resources/cloud_access_policy_token), would these be helpful or is it something else?
